rllib/agents/trainer.py

```diff
@@ -218,8 +218,10 @@
     # Experimental flag to speed up sampling and use "trajectory views" as
     # generic ModelV2 `input_dicts` that can be requested by the model to
     # contain different information on the ongoing episode.
+    # None (default): Automatically determine, whether the API should be used
```

Comment:
Why not enable it by default for PG / A2C globally? Is there any downside?

Reply:
So far I have only fully tested it for `use_lstm=True` (auto-lstm wrapping). I can do: False by default, then PG/PPO/A3C/A2C switch it on (with an info message) iff use_lstm = True AND torch.

Reply:
Hmm, no that's not great, either. If people want to switch it off, they should be able to. Otherwise, if we auto-flip it to True, there is no off-switch anymore if users are using these algos. Leaving it at `None` (default), meaning: detect automatically.